app.py

```diff
@@ -153,9 +188,22 @@ def profile():
     return render_template('/user_management/profile.html', user_list=users)
 
 
-@app.route('/overview')
-def overview():
-    return render_template('overview.html')
+# displays the calendar page
+# calendar uses the "full calendar" api
+@app.route('/calendar')
+def calendar():
+    return render_template("calendar.html")
+
+# This is used so that users can input events into the calender using the 'events.json'
+# It can be adjusted to query data from a db in the next iteration if requested
+@app.route('/data')
+def return_data():
```

Comment:
If you can, try changing /data and return_data() into something more specific, like calendar_data, it might ease confusion in the future.